querybook/webapp/components/AIAssistant/AICommandInput.tsx

```diff
@@ -80,46 +131,52 @@ export const AICommandInput: React.FC<AICommandInputProps> = forwardRef(
                     setNewCommand(newCommand);
                 } else {
                     setCommandValue(value);
+                    setShowCommands(true);
                 }
             },
             [command, commands, setNewCommand, setCommandValue]
         );
 
         const onKeyDown = useCallback(
             (event: React.KeyboardEvent) => {
-                if (filteredCommands.length > 0) {
+                let handled = true;
+                // Cmd + / will reset the command and value and open the command options
+                if (matchKeyMap(event, KeyMap.commandBar.openCommands)) {
+                    setNewCommand(undefined);
+                    setCommandValue('/');
```

Comment:
seems like a pretty dangerous shortcut that would delete the existing text

Reply:
the reason I added this is when to switch commands, it's quite cumbersome to delete all the text and then backspace the command. I guess it's not that easy for user to press this shortcut? or we can think of adding undo in the future.